models/pydantic/models.py

```diff
@@ -6,6 +6,15 @@ class AnimalCreate(BaseModel):
     animal_type: str
     name: str
     birth_date: date
+    breed: str
+    animal_photo: str
+
+
+    @property
```

Comment:
you can use computed fields in pydantic for such cases
https://docs.pydantic.dev/2.0/usage/computed_fields/